AsyncDisplayKit/TextKit/ASTextKitRenderer.mm

```diff
@@ -118,22 +118,6 @@ - (CGSize)size
   return _calculatedSize;
 }
 
-- (void)setConstrainedSize:(CGSize)constrainedSize
```

Comment:
I would need to look back at the changelog, but I believe there was a performance reason why I added the ability to set the constrained size on an existing renderer rather than just recreating the renderer.  It may not be a showstopper though.